README.md

```diff
@@ -22,10 +22,18 @@ If there isn't a theme yet that fits you, creating a new theme is welcome, encou
 
 First create the theme itself, by laying out the theme variables in `./src/site.css`. The top of this file should have every other supported theme listed along with the variables that can and will need to be modified for your theme.
 
-Once the theme is created, ensure to add it as an option in `./public/site.js` within the function `changeTheme` and add a new check in the switch statement. Setting the body to have your theme as an attribute value, and saving the new theme option into the users local storage.
+After creating the colours of your theme you'll need to find what syntax theme matches your new theme the best. The Pulsar Package Frontend uses [`Highlight.js`](https://highlightjs.org/) to handle highlighting code blocks that will appear within packages ReadMes, so you can browse all their available themes on their [website](https://highlightjs.org/static/demo/). Once you've found the one you like, make sure to jot down it's CND URL.
```

Comment:
```suggestion
After creating the colors of your theme you'll need to find what syntax theme matches your new theme the best. The Pulsar Package Frontend uses [`Highlight.js`](https://highlightjs.org/) to handle highlighting code blocks that will appear within package READMEs, so you can browse all their available themes on their [website](https://highlightjs.org/static/demo/). Once you've found the one you like, make sure to jot down its CND URL.
```

As much as I love the correct British spelling I feel this should be in American English :P
Also I think "ReadMes" looks a bit odd and it is usually capitalised.

Reply:
Always does take me a second to realize why my spell check gets frustrated at `colours` but otherwise looks great thanks!